README.md

```diff
@@ -16,3 +18,30 @@ The installation of the package is standard: for details, see GAP reference
 manual, chapter "GAP Packages" (and if you don't have write access to the
 Gap installation, type in GAP "?GAP root directory" to see another relevant
 part of the manual).
+
+## Documentation
+
+Full information and documentation can be found in the manual, available
+as PDF `doc/manual.pdf` or as HTML `htm/chapters.htm`, or on the package
```

Comment:
From which package is this copied - I think I have already seen same corrections needed elsewhere. The path `htm/chapters.htm` is for gapmacro-based manuals.